oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -361,7 +366,9 @@ private HttpResponse getMetadataResponse(
     request.setThrowExceptionOnExecuteError(false);
     HttpResponse response;
     try {
+      LoggingUtils.logRequest(request, SLF4JLOGGER, "auth get metadata sending request.");
       response = request.execute();
+      LoggingUtils.logResponse(response, SLF4JLOGGER, "auth get metadata received response.");
```

Comment:
Is it possible for the request to be more specific for this? Auth sending request for access token, for universe domain, for ...? I think `sending metadata...` is a bit generic and may not be clear as logs

Reply:
Yes, I will take another pass on these messages and try to make them more clear.
Will tag to for another round when I update them.